src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -60,12 +60,12 @@ const populateUISchemaForDisplay = (uiSchema, ionField) => {
   uiSchema.type = display.inputType;
   if (display.inputType === 'radio') {
     uiSchema.options = display.options;
+  } else if (display.inputType === 'select' && !_.isEmpty(display.format) && display.format === 'country-code') {
```

Comment:
if it's a `select` why we need another `else if` here ?

Reply:
updated. thanks